components/widgets/ui/src/main/resources/PhenoTips/Widgets.xml

```diff
@@ -367,6 +382,222 @@
   },
 
   /**
+   * Attaches mousedown observers on newly created suggests.
+   *
+   * @param event the event being observed
+   */
+  onSuggestCreated : function (event) {
+    this.suggestionSelected = false;
+    var _this = this;
+    // A mousedown event occurred in the suggest. Don't want onInputBlurred to perform a search on this loss of focus
+    $(this.container).observe("mousedown", function(event) {
```

Comment:
`this.container` doesn't exist. Are you referring to the suggestions list? That's in `event.memo.container`.